deploy/ods-pipeline/charts/setup/values.yaml

```diff
@@ -0,0 +1,2 @@
+# override name to be consistent with previous, separate chart naming contention(s)
```

Comment:
Typo? should probably chart naming convention(s)

Reply:
Just fixed it 😄